src/pages/project.js

```diff
@@ -45,16 +60,56 @@ export const PageProject = ({ router }) => (
             <Breadcrumbs>
               <ProjectBreadcrumb projectSlug={project.name} />
             </Breadcrumbs>
-            <ProjectDetailsWrapper>
-              <div className="project-details-sidebar">
-                <ProjectDetailsSidebar project={project} />
-              </div>
-              <div className="environments-wrapper">
-                <h3>Environments</h3>
-                {!environments.length && <p>No Environments</p>}
-                <Environments environments={environments} project={project} />
-              </div>
-            </ProjectDetailsWrapper>
+            <ProjectWrapper>
+              <LeftNavTabs activeTab="overview" project={project} />
+              <ProjectDetailsWrapper>
+                <div className="project-details-header">
+                  <ProjectDetailsSidebar project={project} />
+                </div>
+                <div className="environments-wrapper">
+                  <h3>Environments</h3>
```

Comment:
probably don't need this h3 because you've got `production` and `development` environments listed below it

theres also a weirdly large amount of white space between this section and the one above for the overview information

Reply:
https://github.com/uselagoon/lagoon-ui/blob/feature/pr-26/src/styles/pageStyles.js#L263
if you change this to
```
padding: 0px calc((100vw / 16) * 1) 40px calc((100vw / 16) * 1);
```

and this
https://github.com/uselagoon/lagoon-ui/blob/feature/pr-26/src/styles/pageStyles.js#L258
```
padding: 48px calc((100vw / 50) * 1) 0px calc((100vw / 50) * 1);
```

it makes it heaps better, so theres probably other things to modify to clear up the whitespace